src/workloads/execution/UnshardedLookup.yml

```diff
@@ -71,6 +95,38 @@ Actors:
             }}
           ]
         cursor: {batchSize: *batchSize}
+    - OperationMetricsName: MatchAllShardsLookupUnsharded
+      OperationName: RunCommand
+      OperationCommand:
+        aggregate: Collection2
+        pipeline:
+          [
+            {$match: {
+              c: {$in: [84, 112, 253, 396, 495, 515, 638, 754, 872, 998]}
```

Comment:
How do you feel about changing this match stage to something like `{$match: {c: {$lt: 5}}}`? Since the collection is sharded on 'a', we don't need to be so specific about the query in order to target all the shards - unless there was another reason for you to choose this match, in which case let me know!

Reply:
Yes that would be better. I was originally filtering on `a`, and so this was used to target all the shards. I forgot to update the filter when I added `c`.